unfetter-discover/unfetter-ui
# Issue 1085/edit baseline

Requires unfetter-discover/unfetter@1127
Fixes unfetter-discover/unfetter#1085


Baselines can now be edited, though this PR has code which treats the `category` attribute of a `Capability` as an ID instead of category name.  Sample capability data in config/examples/unfetter-stix/assessments3/capabilities.stix.json in the unfetter repo must be updated, and so you'll want to pull https://github.com/unfetter-discover/unfetter/pull/1127.  The `category` attribute will be renamed as well but in a subsequent PR.

There is a bug in how new baselines are saved, in that assessed_object entries for each object assessment are empty.  This results in an empty weighting panel for each capability.  This will be fixed in a subsequent PR as well, as it is not related to loading of a baseline for edit purposes.

## To test:
1. Pull this feature
2. Drop the stix DB
3. Restart the stack
4. Click the down arrow in the sidebar and select `Edit`
5. Verify that the capability group(s) and capability(ies) are listed correctly in the wizard.
6. Exit the wizard and click the down arrow in the sidebar to display the master list of baselines
7. Click the edit button for one of the other baselines
8. Repeat steps 3 through 5
